src/Compilers/Test/Core/Microsoft.CodeAnalysis.Test.Utilities.csproj

```diff
@@ -96,11 +96,13 @@
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net40" Version="$(MicrosoftNETFrameworkReferenceAssembliesnet40Version)" IncludeAssets="none" PrivateAssets="all" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net20" Version="$(MicrosoftNETFrameworkReferenceAssembliesnet20Version)" IncludeAssets="none" PrivateAssets="all" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.NET.Build.Extensions" Version="$(MicrosoftNETBuildExtensionsVersion)" IncludeAssets="none" PrivateAssets="all" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.NETCore.App.Ref" Version="$(MicrosoftNETCoreAppRefVersion)" IncludeAssets="none" PrivateAssets="all" GeneratePathProperty="true" />
     <PackageReference Include="jnm2.ReferenceAssemblies.net35" Version="$(jnm2ReferenceAssembliesnet35Version)" IncludeAssets="none" PrivateAssets="all" GeneratePathProperty="true" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" IncludeAssets="none" PrivateAssets="all" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.CSharp" Version="$(MicrosoftCSharpVersion)" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.VisualBasic" Version="$(MicrosoftVisualBasicVersion)" IncludeAssets="none" PrivateAssets="all" GeneratePathProperty="true" />
+
+    <PackageReference Include="Basic.Reference.Assemblies.NetStandard20" Version="$(BasicReferenceAssembliesNetStandard20Version)" />
```

Comment:
>Basic.Reference.Assemblies.NetStandard20 [](http://example.com/codeflow?start=31&length=40)

How this package is created?

Reply:
For every major target framework there is a NuPkg that contains all of the relevant reference assemblies. For example: 

- [netstandard2.0](https://www.nuget.org/packages/NETStandard.Library/2.0.3)
- [net461](https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies.net461)

The NuPkg all have a very similar structure for how the ref assemblies are laid out. The basic assemblies repo has a [script](https://github.com/jaredpar/basic-reference-assemblies/blob/main/Scripts/Generate.ps1) that just walks that directory structure and generates targets + code that will embed the ref assemblies as resources and expose them as named members on types. 

It's very similar to the way we generate them in our repo but just a bit more polished and generalized.